hoot-cmd/src/main/cpp/hoot/cmd/HelpCmd.cpp

```diff
@@ -5,7 +5,7 @@
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
- *
+ * 
```

Comment:
Deleted the offending whitespace char from the end of line 8 of the LicenseTemplate.txt file. After noticing that the scripts directory had been reorganized, I did a pull to ensure that I had the latest changes. Locally tested UpdateAllCopyrightHeaders.sh in both non-update and update mode. 12 files needed updating. Those same 12 files were updated. All were added, committed and pushed to the 613_3 branch. Should be good-to-go.